crates/uv-requirements/src/source_tree.rs

```diff
@@ -135,7 +135,7 @@ impl<'a, Context: BuildContext> SourceTreeResolver<'a, Context> {
         for name in groups.explicit_names() {
             if !metadata.dependency_groups.contains_key(name) {
                 return Err(anyhow::anyhow!(
-                    "The dependency group '{name}' was not found in {}",
+                    "The dependency group '{name}' was not found in the project at {}",
```

Comment:
Backticks on the path?

Reply:
Oh I assumed we purposefully avoided that in our style so the path was easy to copy/alt-click in terminals

Reply:
We either do `: {path}` or use backticks.